test/clj/game_test/cards/programs.clj

```diff
@@ -515,6 +515,38 @@
       (is (= 4 (:credit (get-corp))) "Corp lost 1 credit to Lamprey")
       (is (= 3 (:credit (get-runner))) "Runner gains 1 credit from Ixodidae due to Lamprey"))))
 
+(deftest kyuban
+  (testing "Basic functionality: Gain 1 counter every Runner turn"
```

Comment:
This testing comment seems misleading.

Reply:
Ooops. Guess where I got this test from.